front-end/e-learning/src/components/Login.css

```
@@ -0,0 +1,7 @@
+.container{
+}
+
+.column{
+    background-color: black;
+    max-width: 50%;
+}
```

put new line at the end of every file.

```suggestion
}

```